openedx/features/pakx/lms/pakx_admin_app/tasks.py

```diff
@@ -99,7 +99,9 @@ def get_formatted_error_msg(errors):
         lang_err_msg = err_map.pop('language_code', None)
         formatted_errors = ['{}:({}) | {}'.format(f, req_data[f], '|'.join(err)) for f, err in err_map.items()]
         formatted_errors += ['language: {}'.format('|'.join(lang_err_msg))] if lang_err_msg else []
-        return '\n'.join(formatted_errors)
+
+        user_key = req_data['email'] or req_data['username'] or profile_req_data.get('name') or index
```

Comment:
please update it to get, get might through an error. i.e req_data['email'] to req_data.get('email')

Reply:
actually, we are manually setting these keys after reading from the CSV file so these keys will always be there.

Reply:
sure